operator/config/e2e-tests/manager.yaml

```diff
@@ -11,9 +11,10 @@ spec:
         imagePullPolicy: IfNotPresent
         args:
         - "--health-probe-bind-address=:8081"
-        - "--metrics-bind-address=127.0.0.1:8080"
+        - "--metrics-bind-address=:8443"
```

Comment:
Why the port change here? Would this conflict with any of stucki's sidecar changes? (I don't think it will but just checking)

Reply:
No it will not conflict